trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/parasite_total.py

```diff
@@ -48,6 +48,8 @@ def parasite_total(state,settings,geometry):
  
     # from fuselage
     for fuselage in fuselages.values():
+        if fuselage.tag == 'fuselage_bwb':
```

Comment:
This was done so that the AVL scripts have a way of determining whether or not to make a tube (for Taw configrations).